app.js

```diff
@@ -1,8 +1,128 @@
-// import functions and grab DOM elements
+const currentPollDiv = document.getElementById('current-poll-div');
+const newPoll = document.getElementById('new-poll');
+const previousPollsDiv = document.getElementById('previous-polls-div');
+const savePollBtn = document.getElementById('save-poll-btn');
 
-// let state
+const optionOnePlus = document.getElementById('option-one-plus-btn');
+const optionOneMinus = document.getElementById('option-one-minus-btn');
+const optionTwoPlus = document.getElementById('option-two-plus-btn');
+const optionTwoMinus = document.getElementById('option-two-minus-btn');
 
-// set event listeners 
-  // get user input
-  // use user input to update state 
-  // update DOM to reflect the new state
+let question = '';
+let optionOne = '';
+let optionTwo = '';
+let votesOne = 0;
+let votesTwo = 0;
+let previousPolls = [];
+
+newPoll.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const data = new FormData(newPoll);
+
+    const userQuestion = data.get('poll-question');
+    const userOptionOne = data.get('option-one-input');
+    const userOptionTwo = data.get('option-two-input');
+
+    question = userQuestion;
+    optionOne = userOptionOne;
+    optionTwo = userOptionTwo;
+
+    renderPoll();
```

Comment:
The renderPoll() function should take 4 arguments based on your definition below, but here you are calling it with 0 arguments.